styles/modules.css

```diff
@@ -56,23 +61,47 @@
 
   #nav li {
     display:inline-block;
-
+    margin: 0 1em;
   }
 
   .icon-menu {
     display: none;
   }
 
-  #articles {
-    width:100%;
-  }
-
   .article {
     margin: 0 auto;
+    margin-top: 2em;
+    width: 100%;
   }
 
   .article p {
   margin:4%;
   }
 
 }
+
+@media screen and (min-width: 1000px){
+
+  .article{
+    max-width:960px;
+    /*this expands div to include image*/
+    overflow: auto;
+  }
+}
+
+
+@media screen and (min-width: 1280px){
+.article{
+    max-width: 1200px;
+    margin-bottom: 6em;
+}
+
+.article img{
+  margin-left: 0;
+  margin-right: 4em;
+  float: left;
+  max-width: 40%;
+  max-height: 100%;
+}
+
+}
```

Comment:
Overall, I like the direction you're going with your portfolio. I know you have an item list of plans to do, so I decided to leave out things like refactoring and styling since you that is what you're going to work on next. I think that's neat you want to include WebDB functionality - I look forward to seeing how that's implemented in your portfolio. You plan on adding an image or some styling in the header - I think this will add quite a bit of depth, aesthetically. The elements on the pages transition smoothly when the viewport changes, great work on RWD! I look forward to seeing your progress as you get these TODO's knocked out! Keep up the good work!